### PR TITLE
chore: update oh.yaml from api+ generated

### PR DESCRIPTION
**Pull request by gasp**
- [x] copy oh.yaml from openhospital-api to openhospital-ui
- [x] generate new api files with `npm run build:api`
- [x] fix conflicts
- [x] test with real api

### CODE REVIEW

File: src/generated/models/PatientDTO.ts [hunk 1]
```diff
@@ -194,46 +194,63 @@ export interface PatientDTO {
...
+    D = 'D',
+    A = 'A'
+}
+/**
+ * @export
+ * @enum {string}
+ * these are manually overwritten 
+ * because openapi-generator does not handle well thoser enums
+ */
+export enum PatientDTOBloodTypeEnum {
```
Comment (gasp) [inline src/generated/models/PatientDTO.ts:222]: this is new

File: src/generated/apis/PermissionsApi.ts [hunk 2]
```diff
@@ -33,58 +25,15 @@ export interface RetrievePermissionByNameRequest {
...
-    id: number;
-    permissionDTO: PermissionDTO;
-}
-
 /**
  * no description
  */
 export class PermissionsApi extends BaseAPI {
 
-    /**
```
Comment (gasp) [inline src/generated/apis/PermissionsApi.ts:50]: This is related to https://github.com/informatici/openhospital-api/pull/462

File: src/generated/models/PatientDTO.ts (hunk 1 above)
Comment (mwithi) [reply src/generated/models/PatientDTO.ts:222]: is this PR applying https://github.com/informatici/openhospital-api/blob/develop/openapi/oh.yaml?

Comment (gasp) [reply src/generated/models/PatientDTO.ts:222]: yes, but not https://github.com/informatici/openhospital-api/pull/468 yet
meanwhile I convert this issue on Draft

Comment (gasp) [reply src/generated/models/PatientDTO.ts:222]: now up to date with the latest changes ✅

File: src/generated/runtime.ts [hunk 3]
```diff
@@ -1,266 +1,211 @@
...
 
-import { Observable, of, Subscriber } from "rxjs";
-import { ajax, AjaxRequest, AjaxResponse } from "rxjs/ajax";
-import { map, concatMap } from "rxjs/operators";
+import { Observable, of, Subscriber } from 'rxjs';
+import { ajax, AjaxRequest, AjaxResponse } from 'rxjs/ajax';
+import { map, concatMap } from 'rxjs/operators';
 
-export const BASE_PATH = "https://oh2.open-hospital.org".replace(/\/+$/, "");
+export const BASE_PATH = 'http://localhost:8080'.replace(/\/+$/, '');
```
Comment (mwithi) [inline src/generated/runtime.ts]: I think this must remain unchanged, correct @SilverD3 ?

Comment (SilverD3) [reply src/generated/runtime.ts]: Yes, sure.

File: src/generated/runtime.ts
Comment (SilverD3) [inline src/generated/runtime.ts:1]: Generally, don't commit this file, unless the configs of openapi-generator have changed.

**Review by SilverD3 — CHANGES_REQUESTED** [review]

File: src/generated/runtime.ts
Comment (gasp) [reply src/generated/runtime.ts:1]: should I revert to previous version?

Comment (mwithi) [reply src/generated/runtime.ts:1]: yes please

Comment (mwithi) [reply src/generated/runtime.ts:1]: I mean, the BASE_PATH only

Comment (gasp) [reply src/generated/runtime.ts:1]: it's done in the other pr: https://github.com/informatici/openhospital-ui/pull/662

**Review by SilverD3 — APPROVED** [review]
Looks good. Let's merge

**Comment by SilverD3** [comment]
@gasp please resolve unresolved conversations

**Review by mwithi — APPROVED** [review]
Reviewed and approved by @SilverD3

**Pull request merged**